### ISSUE TITLE
[gradle] Plugin failure when running a full release

### ISSUE THREAD

**Issue by aalmiray** [jreleaser/jreleaser#1179]
### Task List

- [x] Steps to reproduce provided
- [ ] Stacktrace (if present) provided
- [ ] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

1. Clone https://github.com/onlyf1com/onlyf1-db at https://github.com/onlyf1com/onlyf1-db/commit/4c0bf047ecd61cb8773e628cb972358f966708d2
2. `./gradlew dist assemble jreleaserFullRelease`

### Expected Behaviour

Successful release.

### Actual Behaviour

Build error due to 

```
 FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':jreleaserFullRelease' (type 'JReleaserFullReleaseTask').
  - In plugin 'org.jreleaser' type 'org.jreleaser.gradle.plugin.tasks.JReleaserFullReleaseTask' property 'outputDirectory' specifies directory '/home/runner/work/onlyf1-db/onlyf1-db/build/jreleaser' which doesn't exist.
    
> Task :jreleaserFullRelease FAILED
    Reason: An input file was expected to be present but it doesn't exist.
    
    Possible solutions:
      1. Make sure the directory exists before the task is called.
      2. Make sure that the task which produces the directory is declared as an input.
```

### Environment Information

- **Operating System**: N/A
- **JReleaser Version:** 1.4.0
- **JDK Version:** N/A

cc: @marceloverdijk

**Comment by aalmiray** [comment]
Successfully ran a local release (in dryrun mode) with `v1.4.0`

```
> Task :jreleaserFullRelease
[INFO]  JReleaser 1.4.0
[INFO]    - basedir set to /private/tmp/onlyf1-db
[INFO]    - outputdir set to /private/tmp/onlyf1-db/build/jreleaser
[INFO]  git-root-search set to false
[INFO]  Loading variables from /Users/aalmiray/.jreleaser/config.toml
[INFO]  Validating configuration
[INFO]  Strict mode set to false
[INFO]  Project version set to 2023.0.0.beta1
[INFO]  Release is not snapshot
[INFO]  Timestamp is 2023-01-22T22:07:26.733834+01:00
[INFO]  HEAD is at 54ad78e
[INFO]  Platform is osx-x86_64
[INFO]  dry-run set to true
[INFO]  Generating changelog
[INFO]  Storing changelog: build/jreleaser/release/CHANGELOG.md
[INFO]  Calculating checksums
[INFO]    [checksum] All checksums are up-to-date. Skipping
[INFO]  Signing files
[INFO]    [sign] Signing is not enabled. Skipping
[INFO]  Deploying is not enabled. Skipping
[INFO]  Uploading is not enabled. Skipping
[INFO]  Releasing to https://github.com/onlyf1com/onlyf1-db
[INFO]   - uploading onlyf1-db-csv-2023.0.0.beta1.zip
[INFO]   - uploading onlyf1-db-json-single-2023.0.0.beta1.zip
[INFO]   - uploading onlyf1-db-json-splitted-2023.0.0.beta1.zip
[INFO]   - uploading onlyf1-db-smile-single-2023.0.0.beta1.zip
[INFO]   - uploading onlyf1-db-smile-splitted-2023.0.0.beta1.zip
[INFO]   - uploading onlyf1-db-sqlite-2023.0.0.beta1.zip
[INFO]   - uploading checksums_sha256.txt
[INFO]  Announcing release
[INFO]  Announcing is not enabled. Skipping
[INFO]  Writing output properties to build/jreleaser/output.properties
[INFO]  JReleaser succeeded after 1.145 s
```

There is however a problem in the build script itself that is not necessarily the cause of the issue reported by this ticket.

**Comment by aalmiray** [comment]
Released in v1.5.0 -> https://github.com/jreleaser/jreleaser/releases/tag/v1.5.0
